.github/workflows/build-meson.yml

```diff
@@ -46,11 +46,7 @@ jobs:
         python blint-db/blint_db/cli.py -Z1
         echo $GITHUB_TOKEN | oras login ghcr.io -u $GITHUB_USERNAME --password-stdin
         cd blint-db
-        oras push ghcr.io/appthreat/blintdb-meson:v0.1 \
-          --config ../.oras/config.json:application/vnd.oras.config.v1+json \
-          --annotation-file ../.oras/annotations.json \
-          ./blint.db:application/vnd.appthreat.vdb.layer.v1+tar \
-        
+        echo "$GITHUB_USERNAME;;$GITHUB_TOKEN" | python ./.oras/orasclient.py
```

Comment:
Don't print the tokens on the workflow.

Reply:
what would be a good way to login? as oras does not support `arm64` also some of the packages required for building projects are difficult to find in almalinux.